### DISCUSSION TITLE
[4.1] - "unique" validation on an object at wrong table

### DISCUSSION THREAD

**Discussion by rhadamez** [adonisjs/core#1776] [Help] [answered]
**Hey you!**

I'm making a validation on an object "company" that its table is called "companies", the "unique" validation is looking in the wrong table: "company" instead "companies". I don't know how to procedded.

The image explains the situation:

![toAdonisCommunity](https://user-images.githubusercontent.com/18365154/96634129-7176e380-12f0-11eb-9485-18eebdcff011.png)

Any help would be nice.

**Comment by thetutlage** [comment, marked as answer]
It's looking at the right table but the wrong field. You can define the field name yourself.

```js
unique:companies,cnpj
```

**Reply by rhadamez** [reply]
It works like a charm! thanks a lot.
